plugins/rxp/io/RxpPointcloud.cpp

```diff
@@ -163,9 +166,22 @@ void RxpPointcloud::on_echo_transformed(echo_type echo)
     unsigned int returnNumber = 1;
     for (scanlib::pointcloud::target_count_type i = 0; i < target_count; ++i, ++returnNumber)
     {
-        m_points.emplace_back(targets[i], returnNumber, target_count);
+        m_points.emplace_back(targets[i], returnNumber, target_count,m_edge);
+        if (m_edge)
```

Comment:
A comment explaining that all points after the first aren't marked as EOF might be nice.

A space before `m_edge` above as well.

Reply:
@abellgithub, actually, first point is marked as EOF where it's actually BOF